src/main/twirl/scalajs/jsdeps.scala.html

```diff
@@ -1,3 +1,5 @@
+@import play.twirl.api.TwirlFeatureImports.defining
```

Comment:
`play.twirl.api.TemplateMagic` (where `defining` is defined) is deprecated. Importing `TwirlFeatureImports` avoids compiler warnings and allows scalajs-scripts users to turn on the `-Xfatal-warnings` scalac option.